### DISCUSSION TITLE
鉤織課上大家想學什麼？

### DISCUSSION THREAD

**Discussion by woohsin** [1101-GS4719-C/Python-Community-GS4719-C#1938] [Q&A]
最近在準備一個有關鉤織的課程，大約50分鐘，會有手做跟講解部分，如果有興趣的人會想了解什麼呢？或想實做的東西。

目前是打算稍微介紹一下各種有關的工藝，然後帶大家織基本的針法，當作練習，再進階成實作一個作品。

**Comment by LinChloe** [comment]
應該會想做出些實用的東西 ?
在韓團lesserafim 裡有位女偶像 Sakura，常常會自己織些作品(像是圍巾，帽子，包包)
每個都非常可愛，顏色也很好看~
推薦找一個她的作品教導學員們複製出來。
拿著一個偶像同款，相信學員們也會很開心~
